model/bid.js

```diff
@@ -0,0 +1,94 @@
+// Creates a bid for a user
+const createUserBid = (user, bidAmount, driver, date, time, origin, destination, db) =>  {
+    db.none('INSERT INTO bid(bidder, bidStatus, bidAmount, driver, date, time, origin, destination) ' +
+        'VALUES($1, $2, $3, $4, $5, $6, $7, $8)', [user, 'pending', bidAmount, driver, date, time, origin, destination])
+        .then(() => {
+            console.log("success!");

```

Comment:
Can we change the logs to a specific output? E.g created bid for user 4, amount 450.2